lib/nextstrain/sphinx/theme/layout.html

```diff
@@ -49,17 +45,19 @@
       {%- set nav_version = current_version %}
     {% endif %}
     {% if nav_version %}
-      <span>
-        version: {{ nav_version }}
-      </span>
+      <div class="version">
+        {% if nav_version == "stable" %}
+          {# don't show anything #}
```

Comment:
Could we report version here, too, along with  "stable" to fix [this related issue](https://github.com/nextstrain/augur/issues/817)?

Reply:
_Maybe_? If `version` still retains the actual value when `nav_version` is `latest`, then yes. Reading the [RTD issue](https://github.com/readthedocs/readthedocs.org/issues/4529) linked from our Augur issue, it seems like `version` will retain its value. I can try things out, but will do it on another PR.

Reply:
Trialing things over in https://github.com/nextstrain/sphinx-theme/pull/16